packages/expo-modules-core/ios/FileSystemUtilities/FileSystemUtilities.swift

```diff
@@ -56,6 +56,16 @@ public struct FileSystemUtilities {
     if !permissionForInternalDirs.contains(.none) {
       return permissionForInternalDirs
     }
+    let standardizedPath = path.standardizedFileURL.path
+    if let range = standardizedPath.range(of: "Containers/Shared/AppGroup") {
```

Comment:
i'm sorry for late to review the pr. i have little concern of having this ios internal path here. also that's not ideal to maintain appGroupId from both entitlment and javascript calls. i have some idea to try out. are you open to let me have a pr to work on App Group beforehand (will list you as co-author for sure).